core/transformation/pathfinders/round_1/transform.py

```diff
@@ -161,17 +161,19 @@ def _project_details(
         gis_provided                - from "GIS Provided" in the transformed DF
         lat_long                    - from "Lat/Long" in the transformed DF
     """
-    organisation_name = df_dict["Organisation Name"].iloc[0, 0]
+    organisation_name = df_dict["Organisation name"].iloc[0, 0]
     programme_id = programme_name_to_id_mapping[organisation_name]
-    project_ids = df_dict["Project Location"]["Project name"].map(project_name_to_id_mapping)
-    location_multiplicities = df_dict["Project Location"]["Location"].map(
+    project_ids = df_dict["Project location"]["Project name"].map(project_name_to_id_mapping)
+    location_multiplicities = df_dict["Project location"]['Project full postcode/postcodes (e.g., "AB1D 2EF")'].map(
         lambda x: "Multiple" if "," in x else "Single"
     )
-    locations = df_dict["Project Location"]["Location"].map(lambda x: list(map(str.strip, x.split(","))))
-    postcodes = df_dict["Project Location"]["Location"].map(extract_postcodes)
+    locations = df_dict["Project location"]['Project full postcode/postcodes (e.g., "AB1D 2EF")'].map(
+        lambda x: list(map(str.strip, x.split(",")))
```

Comment:
This is a bit of an unwieldy lambda, and I wonder if it might be more consistent for anything other than a very straightforward lambda to be a function. That way we can properly unit test it and avoid this difficult to read code.

Then the code will read something like:

```locations = df_dict["Project location"]['Project full postcode/postcodes (e.g., "AB1D 2EF")'].map(function)```

Reply:
Hey @Joel-Pearce, while I agree with the principle that anything beyond a simple lambda should be a standalone function, I do think this lambda is quite simple - the lambda itself is just `list(map(str.strip, x.split(",")))` - split the string by commas, strip whitespace and convert to a list. I think the expression as a whole perhaps looks more complex at an initial glance because of the verbosity of the bit that comes before it - `df_dict["Project location"]['Project full postcode/postcodes (e.g., "AB1D 2EF")']`. The other reason I'd rather not put this into a standalone function is that currently in this module we have the one parent, public function `pathfinders_transform`, and then every other function is a "private sub-function" called once by `pathfinders_transform` - separating out this into its own function would break that pattern

Reply:
It feels like something we should test though. Do we throw different strings at this in a test? Obvs in this case it will expensive as a unit test as it has all the extra setup / context

Reply:
also, is this transforming this cell into a list? What is the reason for that as we just capture the original string in the DB for this column